### ISSUE TITLE
Is it possible to support rotation of front / back cameras on a tablet device?

### ISSUE THREAD

**Issue by ckrc** [jacobbo/WebEye#81]
I have integrated WebEye into my WPF app and it works great. However, would be even nicer if rotation of front / back cameras can be supported. Throughts?

**Comment by jacobbo** [comment]
Hello, you mean a button to switch between front a back cameras?

**Comment by ckrc** [comment]
@jacobbo yes, to switch between front and back cameras. Do you think this is possible?

**Comment by jacobbo** [comment]
You just need to stop a stream from a front camera, detach the stream from a window, then start a stream from back camera and attach it to the window.

I do think this functionality should be part of the control as one of the goals is to keep it generic and minimalistic.

**Comment by ckrc** [comment]
@jacobbo That makes sense. Thanks for pointing it out to me. 

Can I ask you about zoom in / out feature. Can this be supported?

**Comment by jacobbo** [comment]
You can easily achieve this using ScaleTransform in WPF, but again, sorry, do not think this should be part of the control.

**Comment by ckrc** [comment]
Thanks @jacobbo 
I will try to implement it myself

**Comment by ckrc** [comment]
Thanks again and I have successfully implemented the zoom feature. @jacobbo
